cblas.h not found when installing on ubuntu 20.04.1

I've had no problem installing amici on my OSX laptop, but no go on my Ubuntu desktop 
Using  the following commands as per the [documentation](https://amici.readthedocs.io/en/v0.11.15/python_installation.html).

```
sudo apt install libatlas-base-dev swig
sudo apt install libhdf5-serial-dev
pip install amici 
```

I get the following inclusion error.

```
 amici/src/cblas.cpp:16:13: fatal error: cblas.h: No such file or directory
     16 |    #include <cblas.h>
        |             ^~~~~~~~~
  compilation terminated.
  error: command '/home/llamagod/miniconda3/envs/rjmcmc_thesis/bin/x86_64-conda-linux-gnu-cc' failed with exit code 1
  ----------------------------------------
  ERROR: Failed building wheel for amici
```

I see that there's an [issue](https://github.com/AMICI-dev/AMICI/issues/781) open talking about making this more flexible.

Is there some kind of fix that I could apply at the moment however?

--- a/documentation/python_installation.rst
+++ b/documentation/python_installation.rst
@@ -388,6 +388,15 @@ install the latter locally, using:
 
    conda install -c conda-forge openblas
 
+To make AMICI use openblas, set the following environment variable:
+
+.. code-block:: bash
+
+   export BLAS_LIBS=-lopenblas
+
+``BLAS_LIBS`` needs to be set during installation of the AMICI package, as
+well as during any future model import.
+
 To install AMICI, now run:
 
 .. code-block:: bash

